src/appMain/life_cycle.cc

```diff
@@ -31,11 +31,11 @@
 */
 #include <csignal>
 
-#include "utils/logger.h"
 #include "./life_cycle.h"
 #include "utils/signals.h"
 #include "config_profile/profile.h"
 #include "resumption/last_state.h"
+#include "utils/logger.h"
```

Comment:
@pestOO, because '<windows.h>' and '<winsock.h>' includes order matter. 'life_cycle.h' and 'logger.h' include them, so we have to include 'logger.h' after 'life_cycle.h'